code/modules/mob/mob.dm

```diff
@@ -978,16 +978,19 @@ GLOBAL_LIST_INIT(slot_equipment_priority, list( \
 			add_spell_to_statpanel(S)
 
 	// Allow admins + PR reviewers to VIEW the panel. Doesnt mean they can click things.
-	if(is_admin(src) || check_rights(R_VIEWRUNTIMES, FALSE))
-		if(statpanel("MC")) //looking at that panel
+	if((is_admin(src) || check_rights(R_VIEWRUNTIMES, FALSE)) && client?.prefs.toggles2 & PREFTOGGLE_2_MC_TABS)
```

Comment:
```suggestion
	if((is_admin(src) || check_rights(R_VIEWRUNTIMES, FALSE)) && (client?.prefs.toggles2 & PREFTOGGLE_2_MC_TABS))
```
bitwise styling

Reply:
I prefer having it like this and its easier to keep track, plus if this went at the start of the `if()` without the brackets they have issues.

I can change it if you really want but I would rather not.

Reply:
Feel free to leave it as is :D